app/pb_kits/playbook/pb_contact/contact.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "action_view"
+
+module Playbook
+  module PbContact
+    class Contact < Playbook::PbKit::Base
+      include ActionView::Helpers::NumberHelper
```

Comment:
I think we should convert this before we even merge this PR

Reply:
We are saving all the conversions for B&B.

Reply:
Once we can update the generators and possibly crack about 50% of the kits, we likely should move all new kit creation into the new format.

Like Jason said though. For now, B&B will keep chipping away one at a time as UX clears out all the leftover Create work and higher priority kit fixes/upgrades like this one (thank you for prioritizing these kits Jason!!!)